src/docs/development/tools/devtools/code-size.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Using the code size tool
+description: Learn how to use the DevTools code size tool.
+---
+
+## What is it?
+
+The code size tool allows you to analyze the total size of your app, including
+both native and Dart code, or just the size of the Dart code in your app. You
```

Comment:
We're pretty firm in the definition here. But we analyze the size of non-code like assets as well.

Reply:
modified